deployer/migrations/0004_change_service_id_to_ulid.sql

```diff
@@ -0,0 +1,169 @@
+-- Copy current service table. Keep the old table around because
+-- the rest of the tables are still having an FK on service_id.
+CREATE TABLE IF NOT EXISTS services_copy (
+    id TEXT PRIMARY KEY,  -- Ulid identifier of the service.
+    uuid TEXT,            -- Old Uuid identifier of the service.
+    name TEXT UNIQUE      -- Name of the service.
+);
+
+INSERT INTO services_copy (id, uuid, name)
+SELECT
+  upper(ulid_with_datetime(strftime('%Y-%m-%d %H:%M:%f', datetime('2023-06-01')))) as id,
```

Comment:
Needed because `ulid_with_datetime` creates ULIDs in lowercase format, while `ulid` crate and the implementation for `Ulid::to_string/from_string` convert them to/from uppercase format. We have to match the case between the two to be able to keep the services state consistent right after the migration.